packages/snaps-rpc-methods/src/restricted/manageState.ts

```diff
@@ -257,10 +257,6 @@ export function getValidatedParams(
     if (!isObject(newState)) {
       throw rpcErrors.invalidParams({
         message: `Invalid ${method} "updateState" parameter: The new state must be a plain object.`,
```

Comment:
```suggestion
        message: `Invalid ${method} "newState" parameter: The new state must be a plain object.`,
```
This seems to be typoed in all of these errors